src/EditorFeatures/VisualBasicTest/CodeActions/UseNamedArguments/UseNamedArgumentsTests.vb

```diff
@@ -260,5 +260,59 @@ Class C
     End Sub
 End Class")
         End Function
+
+        <WorkItem(19175, "https://github.com/dotnet/roslyn/issues/19175")>
+        <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsUseNamedArguments)>
+        Public Async Function TestCaretPositionAtTheEnd1() As Task
+            Await TestInRegularAndScriptAsync(
+"Class C
+    Sub M(arg1 As Integer)
+        M(arg1[||])
+    End Sub
+End Class",
+"Class C
+    Sub M(arg1 As Integer)
+        M(arg1:=arg1)
+    End Sub
+End Class")
+        End Function
+
+        <WorkItem(19175, "https://github.com/dotnet/roslyn/issues/19175")>
+        <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsUseNamedArguments)>
+        Public Async Function TestCaretPositionAtTheEnd2() As Task
+            Await TestInRegularAndScriptAsync(
+"Class C
+    Sub M(arg1 As Integer, arg2 As Integer)
+        M(arg1[||], arg2)
+    End Sub
+End Class",
+"Class C
+    Sub M(arg1 As Integer, arg2 As Integer)
+        M(arg1:=arg1, arg2:=arg2)
+    End Sub
+End Class")
+        End Function
+
+        <WorkItem(19175, "https://github.com/dotnet/roslyn/issues/19175")>
+        <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsUseNamedArguments)>
+        Public Async Function TestCaretPositionAtTheEnd3() As Task
+            Await TestMissingInRegularAndScriptAsync(
+"Class C
+    Sub M(arg1 As Integer, optional arg2 As Integer=1, optional arg3 as Integer=1)
+        M(1,[||],3)
+    End Sub
+End Class")
+        End Function
+
+        <WorkItem(19175, "https://github.com/dotnet/roslyn/issues/19175")>
+        <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsUseNamedArguments)>
+        Public Async Function TestCaretPositionAtTheEnd4() As Task
```

Comment:
My concern with such an approach would be that it could subtly change overload resolution if you now removed an argument.  if there was a two-arg method, with matching names, you'd now be calling that, instead of the original member.